supervisor/docker/addon.py

```diff
@@ -284,7 +308,7 @@ def volumes(self) -> Dict[str, Dict[str, str]]:
         # Init other hardware mappings
 
         # GPIO support
-        if self.addon.with_gpio and self.sys_hardware.support_gpio:
+        if self.addon.with_gpio and self.sys_hardware.helper.support_gpio:
```

Comment:
The new `libgpiod` uses `/dev/gpiochip*` devices. Do we support that? I guess we could just add that as well if an addon wants GPIO access. See e.g. https://embeddedbits.org/new-linux-kernel-gpio-user-space-interface/

Reply:
Done by cgroup rules. We need only to know if we can apply it. But yeah, this function would be gone after I know the sys endpoint and can do it like for audio.